### ISSUE TITLE
Add easyfundraising to websites

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeCENMigration#367]

**Comment by richpomfret** [comment]
To make a mock-up/wireframe so we know where it is going.

**Comment by DeniseColbert** [comment]
Pre-made banners are in the drive:
https://drive.google.com/drive/folders/1Mr1Zp1nSQMd_kp0xxF44jA1HIET6FhbF

**Comment by PatReynolds** [comment]
@DeniseColbert and @PatReynolds to specify placement

**Comment by DeniseColbert** [comment]
The below image is 632 x 316 px

![EF-Joined-Social](https://user-images.githubusercontent.com/19173257/132500673-69627cb5-599b-4455-b828-4828c909a702.png)

This one is 710 x 1000.

![ef-poster-710x1000](https://user-images.githubusercontent.com/19173257/132501025-5f0d92bf-2560-4e24-b023-52edacebcd2f.png)

@AlOneill we are thinking that the easyfundraising image could go in the right-hand section of the Volunteer page (below Donate box) and in Help section below sidebar. Could you mock this up in Test 3 so we can compare them, check accessibility etc. please? 

(the second image isn't ours but we can make similar for our org)

**Comment by AlOneill** [comment]
@PatReynolds @DeniseColbert Those images are too wide to be placed comfortably in a sidebar: 340px is the widest that will work well across all screen and window widths. Portrait orientation would be better than landscape.

I will make a mock-up so that you can see what I mean.

**Comment by AlOneill** [comment]
The first image is ready to view on https://test3.freecen.org.uk/cms/opportunities-to-volunteer-with-freecen/draft-opportunities-to-volunteer-with-freecen

There is currently no visible warning that (the link) will open in a new tab and the style needs tweaking to ensure that the right border (on hover and focus) is visible.

Maybe the resulting text size does remain legible enough? What do you think? Remember to view at all (window) widths from widest down to the single-column of the phone display (phone not needed!).

**Comment by PatReynolds** [comment]
Looks good, Alison - text is legible.

Hmmm.. just went back to really check the small print and the advert has gone!

**Comment by AlOneill** [comment]
Image is there, on test3. Have adjusted (by placing in an islet) so that hover and focus styling look complete.

We will need some text to warn users about opening in a new tab. Anything else?

**Comment by PatReynolds** [comment]
Still not there for me (chrome on windows10:
![image](https://user-images.githubusercontent.com/6338822/133225711-3c753cd4-957c-48cd-bf3e-dbf03f72d099.png)

**Comment by DeniseColbert** [comment]
Not there for me either

![image](https://user-images.githubusercontent.com/19173257/133667168-d579cb7e-9705-4e58-a8b6-ecd003abd8dc.png)

**Comment by DeniseColbert** [comment]
@DeniseColbert to test in Firefox & Edge, @PatReynolds to test in accessibility browser

**Comment by PatReynolds** [comment]
Checked test3 with NVDA using tab to advance, and advert not found.

**Comment by DeniseColbert** [comment]
Checked on FF & Edge and ad is not showing on either one.

**Comment by DeniseColbert** [comment]
I think I've found the problem: Pat went to Test3 live site Volunteer page and I went from the URL in Pat's screenshot. Bottom line: ad **is** there on the DRAFT page: https://test3.freecen.org.uk/cms/opportunities-to-volunteer-with-freecen/draft-opportunities-to-volunteer-with-freecen

**NB**  Small thing, but there's a casing mismatch in the page names/titles:

![image](https://user-images.githubusercontent.com/19173257/136199695-4b8e925c-4c91-4330-b665-1a6238939b10.png)

**Comment by AlOneill** [comment]
Re the "casing mismatch", we decided on sentence case for `h1` headings, hence the change. The page file name stays the same, so there is no issue with link addresses. However, if "volunteer" (here more of a verb than a noun), should be "Volunteer", let me know.

**Comment by DeniseColbert** [comment]
@AlOneill yes please, text for opening in new tab. Additionally, could the ad have a border in FC purple to make it stand out from the background a bit?

**Comment by AlOneill** [comment]
Text and border both added to https://test3.freecen.org.uk/cms/opportunities-to-volunteer-with-freecen/draft-opportunities-to-volunteer-with-freecen 

If they are OK, then the only thing missing is the URL …

**Comment by DeniseColbert** [comment]
URL: https://www.easyfundraising.org.uk/causes/freeukgenealogy/

I've added this issue to the agenda for our documentation meet on Friday, to decide on casing in this instance... almost there with this one!

**Comment by DeniseColbert** [comment]
Volunteer page is ready to be made live. (lower case "v")

@AlOneill to look at placing this in the Help page.

**Comment by AlOneill** [comment]
[Opportunities to volunteer with FreeCEN](https://test3.freecen.org.uk/cms/opportunities-to-volunteer-with-freecen) is now sync-ready.

Will do Help page next.

**Comment by AlOneill** [comment]
@DeniseColbert Have added Easyfundraising to DRAFT Help. Visually, it appears below "Contact us" as asked; currently that puts it last in the source order which might not be as you intended. 

Also, the image becomes unreadable at some screen widths, so setup needs a re-think.

**Comment by DeniseColbert** [comment]
@DeniseColbert to find an image to use on Help page that we can add text alongside.

Source order: leave as is until systemwide source order story is done.

**Comment by DeniseColbert** [comment]
Would this work? https://drive.google.com/file/d/1KlCNoboggjcma1HBB5OUVrqBvTM03LgQ/view?usp=sharing

**Comment by DeniseColbert** [comment]
@AlOneill do you now have access to this image?

**Comment by AlOneill** [comment]
@DeniseColbert Yes, I now have access to the new image. It is significantly wider than the current one. I've placed it on t3 on [DRAFT Help](https://test3.freecen.org.uk/cms/draft-help) for you to consider whether or not it works there, across the range of screen widths.

**Comment by AlOneill** [comment]
@DeniseColbert New image now also placed on t3 on [DRAFT Opportunities to volunteer with FreeCEN](https://test3.freecen.org.uk/cms/draft-opportunities-to-volunteer-with-freecen). It has a little more space on this page — there has always been more space here: I have not changed that.

**Comment by PatReynolds** [comment]
Images are now live on FreeCEN, but there is a possible issue with display. Closing this.   Will open new story if there is an issue.
